presto-druid/src/test/java/io/prestosql/plugin/druid/TestDruidIntegrationSmokeTest.java

```diff
@@ -272,4 +272,17 @@ public void testFilteringForTablesAndColumns()
         actualColumns = computeActual("DESCRIBE " + datasourceB);
         Assert.assertEquals(actualColumns, expectedColumns);
     }
+
+    @Test
```

Comment:
why not to add basic test case to `io.prestosql.plugin.jdbc.TestJdbcIntegrationSmokeTest`?

Reply:
`TestJdbcIntegrationSmokeTest` is not generic
did you mean `AbstractTestIntegrationSmokeTest`?

i thought about that. 
Some connnectors do not support limit pushdown, some support partially (eg mysql & varchars)....
Yet, i am OK if we revisit this later.
would you be ok to coninue with current approach?